drivers/gpu/drm/i915/intel_display.c

```diff
@@ -14075,6 +14075,20 @@ static bool intel_crt_present(struct drm_device *dev)
 	return true;
 }
 
+/*
+ * Workaround for VGA not detected issue on ASUS X441SA
+ */
+static const struct dmi_system_id dp_force_init[] __initconst = {
```

Comment:
Since the callsite (intel_setup_outputs) is not __init, you should not use __initconst here